social-team.md

```diff
@@ -0,0 +1,72 @@
+# Social Team
+
+Node.js as a project has a Social Team, a joint team between the TSC and CommComm to which the project has delegated management of official social media.
+
+## Responsibilities
+
+The Social Team is tasked with maintaining the project social accounts. This includes:
+
+- content creation
+- amplification of our community
+  - sharing, retweeting, and so on
+- engagement with our community
+  - liking, replying, and so on
+- updating account metadata (like bio, avatar, header images, and so on)
+
+With regard to posts and community interaction, there is not requirement or entitlement - rather, posts and interactions are completely at the discretion of the Social Team's members.
```

Comment:
```suggestion
With regard to posts and community interaction, there is no requirement or entitlement - rather, posts and interactions
```

Reply:
@mhdawson I believe your suggestion cut off some of the line. Is this okay?

```suggestion
With regard to posts and community interaction, there is no requirement or entitlement - rather, posts and interactions are completely at the discretion of the Social Team's members.
```

Reply:
yes, sorry missed that it was multi-line. Second time today :(

Reply:
committed!